src/Components/InstanceTypes/SuggestedInstanceTypesTable.js

```diff
@@ -57,8 +78,13 @@ export default function SuggestedInstanceTypesTable() {
                                 <Thead noWrap>
                                     <Tr>
                                         {
-                                            SUGG_INSTANCE_TYPES_TABLE_COLUMNS.map(column => {
-                                                return <Th key={column}>{column}</Th>;
+                                            SUGG_INSTANCE_TYPES_TABLE_COLUMNS.map((column, index) => {
+                                                if (index === 1 || index === 2) {
```

Comment:
@r14chandra how about creating a object for `SUGG_INSTANCE_TYPES_TABLE_COLUMNS` data similar to what we have for [SYSTEM_TABLE_COLUMNS ](https://github.com/RedHatInsights/ros-frontend/blob/main/src/constants.js#L89) and we can add a key for is it is sortable or not. Let me know your thoughts.

Reply:
I think this is fine because the attributes that we had added are not relevant in this case. We just want to iterate on the columns and based on that we want to apply sorting.

Reply:
@r14chandra yes I understand, but I was thinking from readability and dynamic point of view. If we will create object with keys do you think it will help with same?

Reply:
We are using `{columnName}` as the key here so that's fine.

Reply:
Hello @r14chandra, how you implemented it is right. The only thing, I can see that it is little difficult to maintain `index value vs column name` in long run as `index` value might change once anyone updates `SUGG_INSTANCE_TYPES_TABLE_COLUMNS` list. It might lead to mismatch.
 
Will it be possible to declare  `SUGG_INSTANCE_TYPES_TABLE_COLUMNS` differently?
Something like:
  
```
export const SUGG_INSTANCE_TYPES_TABLE_COLUMNS = [
    {
        key: 'instance_type',
        title: 'Suggested instance type',        
        isSortable: true      
    },
     {
        key: 'provider',
        title: 'Provider',        
        isSortable: true      
    },
     {
        key: 'description',
        title: 'Description',        
        isSortable: false      
    },
     {
        key: 'system_count',
        title: 'Systems',        
        isSortable: false      
    }]
```

Reply:
@kgaikwad just to confirm that we want to make use of  `column` instead of `index` for comparison.

Reply:
I agree with the declaring part with @kgaikwad here and I understand that the [sortBy prop](https://react-staging.patternfly.org/components/table#isortby) needs an index. My suggestion will be to use f[indIndex array method](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/findIndex) to find the key based on the index and same way we can pass the key the method for API call for params. 
Please check above suggestions [here](https://github.com/RedHatInsights/ros-frontend/pull/292/files#r1403004117) and [here](https://github.com/RedHatInsights/ros-frontend/pull/292/files#r1403000007) for more info and let me know if you have any doubts around it 🙌🏻

Reply:
> @kgaikwad just to confirm that we want to make use of `column` instead of `index` for comparison.

@r14chandra, no need of index comparison. 
- You can make use of `isSortable` value while adding `sort` attribute.
- `getSortParams(index)` you will already have `index` value.

I haven't tried it but you can modify existing iteration something like -  

```
{
  SUGG_INSTANCE_TYPES_TABLE_COLUMNS.map((column_info, index) => {
    const th_attributes = {
       key=column_info.key
    }

    if (column_info.isSortable) {
      th_attributes.sort = getSortParams(index);
    }

    return <Th { ...th_attributes }>{column_info.title}</Th>;
  })
}

```

OR

```
{
  SUGG_INSTANCE_TYPES_TABLE_COLUMNS.map((column_info, index) => {
      return <Th  key={column_info.key} ...(column.isSortable && { sort: getSortParams(index) } )>{column_info.title}</Th>;
  })
}


```